src/App/Settlements/sagas.ts

```diff
@@ -519,7 +555,10 @@ function* validateSettlementReport(): any {
   const adjustments = buildAdjustments(report, finalizeData);
 
   const [debits, credits] = adjustments.reduce(
-    ([dr, cr], adj) => (adj.amount < 0 ? [[...dr, adj], cr] : [dr, [...cr, adj]]),
+    ([dr, cr], adj) =>
```

Comment:
This change was delivered as part of https://github.com/mojaloop/reporting-hub-bop-settlements-ui/pull/19, please pull and update your PR.